### DISCUSSION TITLE
Is there a variable to know if the ignition switch is on?

### DISCUSSION THREAD

**Discussion by wileyknight** [kutu/pyirsdk#48] [General]
I am using a a physical toggle up and down switch to turn on and off the ignition switch. In software this is controlled by a momentary press button. Is there a variable I can access that will tell me if the ignition switch is off in the car that I can use to tell my dashboard to send a pulse if needed? I tried to use the voltage but I am not getting a reading from that var.

**Comment by kutu** [comment]
try `dcStarter` `ManifoldPress` `OilPress`
maybe check if rpm close to 0 = car turned off

**Reply by wileyknight** [reply]
I didn't think about that. Those are good suggestions. Thank you.
